spec/acceptance/api/signup_result_with_access_token_spec.rb

```diff
@@ -10,7 +10,7 @@
     result.save
     result
   end
-  let(:expected_account_properties) { %w[id created_at updated_at admin_domain domain from_email state] }
+  let(:expected_account_properties) { %w[id created_at updated_at domain from_email state] }
```

Comment:
same, why remove this?

Reply:
Same, because the test fails now that admin_domain is private.
😄